### ISSUE TITLE
[PT] Update lesson: Benchee

### ISSUE THREAD

**Issue by thiamsantos** [elixirschool/elixirschool#2398]
## Motivation

The portuguese translation for the lesson [Benchee](https://elixirschool.com/en/lessons/libraries/benchee/) needs to be updated.
We need to update the version of the lesson from `1.0.1` to `2.0.0`.

## Proposed Solution

Translate the updates of the lesson.

You can run `git diff` to see all differences between the versions of the lesson:

```sh
git diff 15547012c9e0e6a8200736d71e5ad1593bddf3b5 HEAD en/lessons/libraries/benchee.md
```

## Tips

- Check the [contributing guidelines](https://github.com/elixirschool/elixirschool/blob/master/CONTRIBUTING.md) before starting.
- Feel free to open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/#tag-your-work-in-progress) as soon as you start working on the translation. In order to prevent we prevent having multiple people working on the translation and also allow the reviewers to give early feedback.
- In case of any doubt feel free to comment in this issue.

**Comment by herminiotorres** [comment]
I take this update lesson

**Comment by thiamsantos** [comment]
Closed by #2410 Thanks @herminiotorres
